avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -100,6 +104,24 @@ def extractMBInfo(avaDir):
             log.info('Saved to dfa_mass_balance/%s ' % (os.path.basename(savename)))
             countFile = countFile + 1
 
+    return countFile
+
+def getMBInfo(avaDir, countStart=0):
+    """ Get MB info """
+
+    # Get info from ExpLog
+    mbFiles = glob.glob(os.path.join(avaDir, 'Outputs', 'com1DFAPy', 'mass*.txt'))
```

Comment:
Could we use the same strategy as with eg. the cfgUtils and use the modules name to get the correct folder -> would make the function more generic. 
I am aware oft the problem with com1DFAPy also being called com1DFA, but we could put this behind a TODO/Dev flag and remove it when we move com1DFAPy...